README.md

```diff
@@ -113,3 +135,40 @@ Client Implementation
 
 `pint`, our Python-based client, is available in the Public Cloud Module, or in
 the Cloud:Tools project in OBS, as `python-susepubliccloudinfo`.
+
+Development
+-----------
+The following steps are only recommended if you want to build this project from sources, work on the codebase or test the latest development changes
+
+  1. **Install Git**
+
+         $ sudo zypper in git
+
+  2. **Install basic Ruby environment**
+
+         $ sudo zypper in ruby rubygem-bundler
```

Comment:
as an FYI, rubygem-bundler wasnt found for me on 15.2 , though "bundle" below worked find regardless